src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -294,7 +294,7 @@ def load_command_table(self, _):
     with self.command_group('functionapp vnet-integration') as g:
         g.custom_command('add', 'add_functionapp_vnet_integration', validator=validate_add_vnet, exception_handler=ex_handler_factory())
         g.custom_command('list', 'list_vnet_integration')
-        g.custom_command('remove', 'remove_vnet_integration')
+        g.custom_command('remove', 'remove_functionapp_vnet_integration')
```

Comment:
Alternatively, I think we can use a `validator` here like the one used for `functionapp vnet-integration add`.

Reply:
That way, we don't have to for example, also rewrite the `list_vnet_integration` function. We can just add the `validator` that will apply to both of these functions (and maybe other functions that we might not be aware of at the moment).

Reply:
I don't understand why we need to rewrite `list_vnet_integration` without validator?

Reply:
I was thinking should `list_vnet_integration` command just work as normal, since it just querying no updates to the vnet. I will confirm with the PM and let you kno.